<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winpcapClass</class>
 <widget class="QMainWindow" name="winpcapClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>winpcap</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>网卡列表</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="serchButton">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>531</width>
      <height>181</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>来源ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源Mac</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标Mac</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>协议</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>包长</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标端口</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>258</width>
      <height>212</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据包详情：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>捕获过滤设置</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>网卡详细信息显示：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="addrEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>461</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>选择协议</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="filterComboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>最大包设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="limitSpinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>listView</zorder>
   <zorder>tableWidget</zorder>
   <zorder>runButton</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>过滤</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="捕获过滤">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="捕获过滤">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>捕获过滤</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="winpcap.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>serchButton</sender>
   <signal>clicked()</signal>
   <receiver>winpcapClass</receiver>
   <slot>serchButton_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>serchButton_Click()</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
